Formula/b/build2.rb

```diff
@@ -0,0 +1,106 @@
+class Build2 < Formula
+  desc "C/C++ Build Toolchain"
+  homepage "https://build2.org"
+  url "https://download.build2.org/0.16.0/build2-toolchain-0.16.0.tar.xz"
+  sha256 "23793f682a17b1d95c80bbd849244735ed59a3e27361529aa4865d2776ff8adc"
+  license "MIT"
+
+  # For background, see https://download.build2.org/README
+  #
```

Comment:
Let's just remove this comment, it doesn't make anything clearer for me.

Reply:
Removed.